unit/Makefile

```diff
@@ -1,19 +1,14 @@
-SRC = cpp_parser.cpp \
-      cpp_scanner.cpp \
-      elf_reader.cpp \
-      float_utils.cpp \
-      ieee_float.cpp \
-      json.cpp \
+CATCH_TESTS = catch_example.cpp \
+              util/sharing_map.cpp \
+              #Empty last line
+
+SRC = catch_entry_point.cpp \
       miniBDD.cpp \
-      osx_fat_reader.cpp \
-      sharing_map.cpp \
       sharing_node.cpp \
-      smt2_parser.cpp \
       string_utils.cpp \
-      wp.cpp \
       # Empty last line
 
-INCLUDES= -I ../src/
+INCLUDES= -I ../src/ -I.
```

Comment:
I thought this is needed so that include paths can be relative to the root of the unit directory, is that not true?